data/json/monsters/mechsuits.json

```
@@ -0,0 +1,139 @@
+[
+  {
+    "id": "mon_mech_recon",
+    "type": "MONSTER",
+    "name": "X-03: 'Spectre' recon mech",
```

```suggestion
    "name": "X-03: 'Spectre' Recon Mech",
```
All caps, like in combat and lifting mechs? Or vice versa, combat and lifting mechs should be decapitalized?